.circleci/config.yml

```diff
@@ -36,3 +38,14 @@ workflows:
           filters:
             branches:
               only: production
+
+      - silta/drupal-build-deploy:
```

Comment:
Why do we need this part - if there are no rules that apply to branch it will simply be not deployed and that is done already in prev. lines.
Regarding ease of turning back on - isn't deleting single line (30) is easier than understanding what this block does  ?

Reply:
I agree with @gatis

Reply:
We still want to run build & tests on Dependabot branches. That's what this workflow is for.

Reply:
Agreed. Lets still build it as it will give some indication to devs as build could still fail if some dependencies could not be met.